x-pack/plugins/uptime/public/components/monitor/status_details/location_map/embeddables/map_config.test.ts

```diff
@@ -4,10 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { getLayerList } from '../map_config';
-import { mockLayerList } from './__mocks__/poly_layer_mock';
```

Comment:
This file could stay in `__mocks__`. There is some special handling around mocks, like with code coverage reports.

Reply:
Alright, done! 👍